Minor changes to nutraceuticals (#798)
greenelab/covid19-review: content/21.nutraceuticals.md

It affects inflammation by modulating cytokine production, decreasing histamine levels, enhancing the differentiation and proliferation of T- and B-lymphocytes, increasing antibody levels, and protecting against the negative effects of reactive oxygen species amongst other effects related to COVID-19 pathology [@doi:10.1155/2014/426740; @doi:10.1007/s00210-013-0880-1; @doi:10.3390/nu9111211].
Vitamin C is utilized by the body during viral infections, as evinced by lower concentrations in leukocytes and lower concentrations of urinary vitamin C.
Post-infection, these levels return to baseline ranges [@doi:10.1177/003693307301800102; @doi:10.1111/j.1749-6632.1975.tb29312.x; @doi:10/fd22sv; @doi:10.3390/nu9040339; @doi:10.1079/bjn19920004].

> It has been shown that as little as 0.1 g/d vitamin C can maintain a normal plasma levels of vitamin C in healthy individuals but that higher doses of at least 1-3 g/d are required for those who are critically ill in ICUs [@doi:10.3390/nu11040708].

Reviewer: ```suggestion
It has been shown that as little as 0.1 g/d of vitamin C can maintain normal plasma levels of vitamin C in healthy individuals, but higher doses of at least 1-3 g/d are required for critically ill patients in ICUs [@doi:10.3390/nu11040708].
```

Even though evidence supporting the use of vitamin C is beginning to emerge, we will not know how effective vitamin C is as a therapeutic for quite some time.
Currently (as of January 2021) over fifteen trials are registered with clinicaltrials.gov that are either recruiting, active or are currently in preparation.
When completed, these trials will provide crucial evidence on the efficacy of vitamin C as a therapeutic for COVID-19 infection.
However, the majority of supplementation studies investigate the intravenous infusion of vitamin C in severe patients. 
Therefore, there is a lack of studies investigating the potential prophylactic administration of vitamin C via oral supplementation for healthy individuals or potentially asymptomatic SARS-CoV-2 positive patients.
Once again, vitamin C intake is part of a healthy diet and the vitamin likely presents minimal risk, but its potential prophylactic or therapeutic effects against COVID-19 are yet to be determined.
To maintain vitamin C status, it would be prudent for individuals to ensure that they consume the recommended dietary allowance (RDA) of vitamin C to maintain a healthy immune system [@doi:10.3390/nu12051466]. 

> The RDA recommended by the FDA is 75-90 mg/d and EFSA recommends 110 mg/d [@doi:10.17226/9810].

Reviewer: ```suggestion
The RDA recommended by the FDA is 75-90 mg/d, and EFSA recommends 110 mg/d [@doi:10.17226/9810].
```

In Slovenia, doctors have been advised to provide nursing home patients with vitamin D [@doi:10.1136/bmj.m1820].

> Despite vitamin D deficiency being a widespread issue in the United States [@doi:10.7759/cureus.2741] the National Institutes of Health have stated that there is "insufficient data to recommend either for or against the use of vitamin D for the prevention or treatment of COVID-19" [@url:https://www.covid19treatmentguidelines.nih.gov/adjunctive-therapy/vitamin-d/].

Reviewer: ```suggestion
Despite vitamin D deficiency being a widespread issue in the United States [@doi:10.7759/cureus.2741], the National Institutes of Health have stated that there is "insufficient data to recommend either for or against the use of vitamin D for the prevention or treatment of COVID-19" [@url:https://www.covid19treatmentguidelines.nih.gov/adjunctive-therapy/vitamin-d/].
```

These are just some examples of how public health guidance has responded to the emerging evidence with regards to vitamin D and COVID-19.

> There is also evidence that individuals are paying attention to their seasonal vitamin D levels, as a survey of polish consumers showed that 56% of respondents used vitamin D during the pandemic [@doi:10.3390/nu13010054].  

Reviewer: ```suggestion
There is also evidence that individuals may be paying attention to their vitamin D levels, as a survey of Polish consumers showed that 56% of respondents used vitamin D during the pandemic [@doi:10.3390/nu13010054].  
```
I'm not sure if seasonal totally fits here since it doesn't seem like they analyzed whether interest changed over different seasons?

Author: I see what you are saying, it was not that seasonality was investigated, it was that the consumers responded in the survey that they took these seasonally... but due to the pandemic the interest in taking these increased. I think that is confusing, so we can leave it out as the meaning is still retained e.g. there was an increase in consumption

> However, some companies have used the emerging evidence surrounding vitamin D to sell products that claim to prevent and treat COVID-19, which in one incident required a federal court to intervene and issue an injunction barring the sale of vitamin-D-related products, due to a lack of clinical data to support their claims [@url:https://www.justice.gov/opa/pr/court-orders-georgia-defendants-stop-selling-vitamin-d-products-treatments-covid-19-and-other]

Reviewer: ```suggestion
However, some companies have used the emerging evidence surrounding vitamin D to sell products that claim to prevent and treat COVID-19, which in one incident required a federal court to intervene and issue an injunction barring the sale of vitamin-D-related products due to the lack of clinical data supporting these claims [@url:https://www.justice.gov/opa/pr/court-orders-georgia-defendants-stop-selling-vitamin-d-products-treatments-covid-19-and-other]
```

Melatonin is another supplement that has been identified as a potential antiviral agent against SARS-CoV-2 using computational methods [@doi:10.1038/s41421-020-0153-3], and it has also been highlighted as a potential therapeutic agent for COVID-19 due to its documented antioxidant, anti-apoptotic, immunomodulatory, and anti-inflammatory effects [@doi:10.3389/fnut.2020.583080; @doi:10.3390/antiox10010047; @doi:10.3390/nu13010047].
Notably, melatonin, vitamin D and zinc were included in the treatment plan of the Former President of the United States upon hospitalization due to COVID-19 [@doi:10.1126/science.abf0974].
These are just some of the many substances and supplements under investigation that currently have limited evidence to support their use or supplementation with the aim to prevent or treat COVID-19.

> While there is plenty speculation surrounding the use of supplements by physicians and scientists, this has not stopped consumers from purchasing these products as one study in Poland has reported that searches for dietary supplements trended online at the start of the pandemic. 

Reviewer: ```suggestion
While there is plenty of skepticism put forward by physicians and scientists surrounding the use of supplements, these statements have not stopped consumers from purchasing these products, with one study reporting that online searches for dietary supplements in Poland began trending with the start of the pandemic [@doi:10.3390/nu13010054].
```

> Additionally, supplement usage increased between the first and second wave of the pandemic . 

Reviewer: I would recommend adding the citation in again!

Author: I think you put it in, do you want to add another reference to it? @rando2

Reviewer: Oops! I didn't realize this was also the Polish study. In that case, I'll just suggest removing the final space!

```suggestion
Additionally, supplement usage increased between the first and second wave of the pandemic. 
```

> Particpants reported various reasons for their use including to improve immunity (60%), to improve overall health (57%), and to fill nutrient gaps in their diet (53%). 

Reviewer: ```suggestion
Particpants reported various reasons for their use of suppelements, including to improve immunity (60%), to improve overall health (57%), and to fill nutrient gaps in their diet (53%). 
```

> Particpants reported various reasons for their use of suppelements, including to improve immunity (60%), to improve overall health (57%), and to fill nutrient gaps in their diet (53%). 

Reviewer: ```suggestion
Participants reported various reasons for their use of supplements, including to improve immunity (60%), to improve overall health (57%), and to fill nutrient gaps in their diet (53%). 
```
